.dockerignore

```
@@ -1,40 +1,72 @@
-# Git
-.git
-.gitignore
+# -------------------------------------------------
+# Base: reuse patterns from .gitignore
+# -------------------------------------------------
+
+# Operating-system
+.DS_Store
+Thumbs.db
+
+# Editor / IDE settings
+.vscode/
+!.vscode/launch.json
```

```suggestion
.vscode/
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
!.vscode/*.code-snippets
```

(stolen from https://www.toptal.com/developers/gitignore/api/visualstudiocode but I like this)

---

I think we can add this when relevant

---

Let's add this now, many users use vscode features for local developement.